besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1894,6 +1899,20 @@ public void validateRpcOptionsParams() {
     }
   }
 
+  public void validateChainDataPruningParams() {
+    if (Boolean.TRUE.equals(unstableChainDataPruningOptions.getChainDataPruningEnabled())) {
+      if (unstableChainDataPruningOptions.getChainDataPruningBlocksRetained()
+          < ChainDataPruningOptions.DEFAULT_CHAIN_DATA_PRUNING_MIN_BLOCKS_RETAINED) {
+        throw new ParameterException(
+            this.commandLine, "--Xchain-data-pruning-blocks-retained must be >= 50400");
```

Comment:
nit: Append `ChainDataPruningOptions.DEFAULT_CHAIN_DATA_PRUNING_MIN_BLOCKS_RETAINED` to exception message instead of literal `50400`.